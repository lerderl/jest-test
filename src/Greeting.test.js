import { render, screen } from "@testing-library/react";

import Greeting from "./Greeting";

describe('Greeting', () => {
  it('will match snapshot with null', () => {
    var { component } = render(<Greeting />);
    expect(component).toMatchSnapshot();
  });

  it('will match snapshot with name', () => {
    const name = "Joe";
    var { component } = render(<Greeting name={name} />);
    screen.debug();
    expect(component).toMatchSnapshot();
  });
});
