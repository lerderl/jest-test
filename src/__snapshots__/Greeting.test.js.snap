// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting will match snapshot with name 1`] = `undefined`;

exports[`Greeting will match snapshot with null 1`] = `undefined`;
